packages/markdown-docx/src/ToCiceroMarkVisitor.js

```diff
@@ -334,14 +442,41 @@ class ToCiceroMarkVisitor {
                         if (runTimeProperties.attributes['w:fill'] === 'F9F2F4') {
                             shadeCodePresent = true;
                         }
+                    } else if (runTimeProperties.name === 'w:vanish') {
+                        vanishPropertyPresent = true;
+                    } else if (runTimeProperties.name === 'w:rFonts') {
+                        if (runTimeProperties.attributes['w:ascii'] === 'Baskerville Old Face') {
```

Comment:
Why is there a different font family for Optional?

Reply:
@algomaster99 
simply using `w:vanish` we cant tell whether this is of `whenSome` type or `whenNone`. So, if 
```
vanish+font=whenNone
vanish=whenSome
```

As for the different font family, I did not want to make it similar to the existing ones or the one used in `code` or `codeblock`.

Reply:
I am not sure if it's true and it's merely based on observation. I have noticed that `whenNone` is always an empty array in case of `Optional`.

Reply:
@algomaster99 
It is not true. It can contain some nodes as visible [here](https://docs.accordproject.org/docs/markup-templatemark.html#optional-blocks). When this is pasted in https://templatemark-dingus.netlify.app/, whenNone can have some nodes.